### DISCUSSION TITLE
关于 - Reverier's Blog

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#142] [Announcements]
# 关于 - Reverier's Blog

一点简介 常用ID是 Reverier-Xu, 不过大家都会简化成RX, 或者叫 R叉 (💢) ; 在 西安电子科技大学 学 网络空间安全 (研究生在读); 写过一点C++, 写过一点go,

https://blog.woooo.tech/about/

**Comment by tgf668** [comment]
围观大佬🤤
